.github/workflows/build.yml

```diff
@@ -54,6 +59,16 @@ jobs:
           command: test
           args: --all-features --workspace
 
+      - name: Nightly Test
+        uses: actions-rs/cargo@v1
+        env:
+          RUSTFLAGS: '--cfg nightly -Zcrate-attr=feature(variant_count)'
+        with:
+          toolchain: nightly
+          command: test
+          args: nightly
```

Comment:
What does this do?

Reply:
you mean the `nightly` args? It's just so we filter and only run the tests with `nightly` on the name. I tried to think of a way to use the `#[cfg(nightly)]` but couldn't come with a way.

Reply:
Ah right. Can't we just run all tests? What is the issue with that? :)

Reply:
Just redundancy, do you suggest then that we only run tests once and with nightly?

Reply:
I don't think CI time matters much in this repository. In other words, I would assume running tests twice is not an issue.